lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Card = props => {
   return (
     <div className="card">
-      <div className="headline">{/* headline goes here */}</div>
+      <div className="headline">{props.card.headline}</div>
       <div className="author">
         <div className="img-container">
-          <img src={'' /* image source goes here */} />
+          <img alt='img' src={props.card.img} />
         </div>
-        <span>By {/* author goes here */}</span>
+        <span>By {props.card.author}</span>
       </div>
     </div>
   );
 };
 
 // Make sure to include PropTypes.
 
-export default Card;
+
+Card.propTypes = {
+  card: PropTypes.shape({
+    author: PropTypes.string,
+    headline: PropTypes.string,
+    img: PropTypes.string,
+    tab: PropTypes.string
```

Comment:
Good work using shape here. Don't forget to add `.isRequired` to anything that should be required